recipes/julia/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "julia" %}
+{% set version = "0.5.2" %}
+{% set sha256 = "efa3303d63e39bf3ea06e23ef68807cca0095c7e0507bdf26eae1d5dee7868c7" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/JuliaLang/julia/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [osx or win]
+  number: 0
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - gcc  # [linux]
+    - toolchain
+    - patchelf
+    - cmake
+    - zlib 1.2.8
+    - python 2.7.*
+    - autoconf 2.69 pl5.*
+    - automake 1.15 pl5.*
+    - openblas 0.2.19|0.2.19.*
+    - blas 1.1 {{ variant }}
+    - fftw
+    - gmp 6.1.*
+    - mpfr 3.1.*
+    - curl
+    - libssh2 1.7.*
```

Comment:
So are you suggesting to add `libssh2` as a runtime requirement?

It looks like the `pkg` tests pass.

```
     * pkg                  INFO: Initializing package repository /tmp/Zpu2ogMF/v0.5
INFO: Cloning METADATA from https://github.com/JuliaLang/METADATA.jl
INFO: No packages to install, update or remove
INFO: Cloning cache of Example from notarealprotocol://github.com/JuliaLang/Example.jl.git
INFO: Cloning cache of Example from https://github.com/JuliaLang/Example.jl.git
INFO: Installing Example v0.4.1
INFO: Package database updated
INFO: Checking out Example master...
INFO: Pulling Example latest master...
INFO: No packages to install, update or remove
INFO: Freeing Example
INFO: No packages to install, update or remove
INFO: Checking out Example master...
INFO: Pulling Example latest master...
INFO: No packages to install, update or remove
INFO: Freeing Example
INFO: No packages to install, update or remove
INFO: Removing Example v0.4.1
INFO: Package database updated
INFO: Nothing to be done
INFO: Cloning Example from https://github.com/JuliaLang/Example.jl.git
INFO: Computing changes...
INFO: No packages to install, update or remove
INFO: Package database updated
INFO: Freeing Example
INFO: No packages to install, update or remove
INFO: Checking out Example master...
INFO: Pulling Example latest master...
INFO: No packages to install, update or remove
INFO: Freeing Example
INFO: No packages to install, update or remove
INFO: Cloning Example2 from /tmp/Zpu2ogMF/v0.5/Example
INFO: Computing changes...
INFO: No packages to install, update or remove
INFO: Cloning Example3 from /tmp/Zpu2ogMF/v0.5/Example
INFO: Computing changes...
INFO: No packages to install, update or remove
INFO: Checking out Example2 test-branch-1...
INFO: Pulling Example2 latest test-branch-1...
INFO: No packages to install, update or remove
INFO: Checking out Example3 test-branch-1...
INFO: Pulling Example3 latest test-branch-1...
INFO: No packages to install, update or remove
INFO: Checking out Example master...
INFO: Pulling Example latest master...
INFO: No packages to install, update or remove
INFO: Cloning Example4 from /tmp/Zpu2ogMF/v0.5/Example
INFO: Computing changes...
INFO: No packages to install, update or remove
INFO: Checking out Example4 test-branch-2...
INFO: Pulling Example4 latest test-branch-2...
INFO: No packages to install, update or remove
INFO: Updating METADATA...
INFO: Updating Example2 test-branch-1...
INFO: Updating Example4 test-branch-2...
INFO: Updating Example3 test-branch-1...
INFO: Computing changes...
INFO: No packages to install, update or remove
INFO: Removing Example v0.4.1
INFO: Package database updated
INFO: Freeing Example
INFO: Upgrading ColorTypes: v0.2.12 => v0.5.2
INFO: Upgrading Colors: v0.6.4 => v0.7.4
INFO: Upgrading FixedPointNumbers: v0.2.1 => v0.3.9
INFO: Removing Example v0.4.1
INFO: Package database updated
WARNING: FixedPointNumbers is fixed at 0.0.6+ conflicting with requirement for ColorTypes: [0.3.0,∞)
WARNING: FixedPointNumbers is fixed at 0.0.6+ conflicting with requirement for Colors: [0.3.0,∞)
WARNING: ColorTypes is fixed at 0.0.0- conflicting with requirement for Colors: [0.3.0,∞)
WARNING: Compat is fixed at 0.2.12+ conflicting with requirement for FixedPointNumbers: [0.17.0,∞)
WARNING: Compat is fixed at 0.2.12+ conflicting with requirement for ColorTypes: [0.26.0,∞)
WARNING: Compat is fixed at 0.2.12+ conflicting with requirement for Colors: [0.18.0,∞)
INFO: Installing Example v0.4.0
INFO: Package database updated
INFO: METADATA is out-of-date — you may not have the latest version of Example
INFO: Use `Pkg.update()` to get the latest versions of your packages
 in 240.46 seconds, maxrss  434.51 MB
     * libgit2-online        in   0.47 seconds, maxrss  434.51 MB
    SUCCESS
```